translations/handleDesktopTranslations.sh

```diff
@@ -12,16 +12,58 @@ gpg --list-keys
 git clone git@github.com:nextcloud/desktop.git
 cd desktop
 
-lupdate-qt5 src/gui/ src/cmd/ src/common/ src/crashreporter/ src/csync/ src/libsync/ -ts translations/client_en.ts
+# Generate source translation files for master and stable branches
+mkdir /branches
+
+versions="stable-2.6 master"
```

Comment:
Do not hardcode this, but instead use git to generate the list:
versions="master $(git branch -r | grep "origin/stable" | cut -f2 -d"/")"

Reply:
Then it should probably match a bit more in detail on `/^stable-[0-9]+(\.[0-9]+)?$/` (untested regex) since otherwise we also try pushing translations if someone pushes a random branch starting with stable.

Reply:
True, but luckily this did not happened yet (on Android where I have this from)

Reply:
I tried the dynamic approach before hardcoding but it lead to errors because we have a lot of old stable branches (stable-2.3.x.x) there were missing some src/xyz folders thus causing the Qt5 tools to fail.
Then I liked the idea of the .tx/backport thing because this would allow us to specify the branches in the repo, giving us full control (ant not to this script here xD).
The hardcode here would only be the fallback then.

Reply:
It is up to you, but I guess the regex approach from julius would also fit.
Either way, I would only go with one approach, to not complicate things.

Reply:
I played a bit with grep regex to satisfy our current stable branch name scheme requirements:

- stable-2.6
- stable-2.7

Thanks for the starting point ;)